rust/agents/relayer/src/settings/matching_list.rs

```diff
@@ -189,39 +192,69 @@ struct WhitelistElement {
     destination_address: Filter<H256>,
 }
 
-impl Display for WhitelistElement {
+impl Display for ListElement {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
         write!(f, "{{sourceDomain: {}, sourceAddress: {}, destinationDomain: {}, destinationAddress: {}}}", self.source_domain, self.source_address, self.destination_domain, self.destination_address)
     }
 }
 
-impl Whitelist {
+#[derive(Copy, Clone, Debug)]
+struct MatchInfo<'a> {
+    src_domain: u32,
```

Comment:
nit: noticing that here we do `src_domain` and in `ListElement` we do `source_domain` (& similar patterns for the other struct fields). I think the field names should be consolidated to try to use consistent names